metashare/api/models.py

```diff
@@ -189,35 +200,119 @@ class Project(mixins.HashIdMixin, mixins.TimestampsMixin, SlugMixin, models.Mode
     def __str__(self):
         return self.name
 
+    def subscribable_by(self, user):  # pragma: nocover
+        # TODO: revisit this?
+        return True
```

Comment:
Right now this is only being used to live-update the `branch_url` while creating a scratch org... but I think it's okay?